docs/src/dev/developer/for-committers.asciidoc

```diff
@@ -141,8 +141,8 @@ TinkerPop has several release branches:
 * `3.4-dev` - 3.4.x (no longer maintained)
 * `3.5-dev` - 3.5.x (non-breaking bug fixes and enhancements)
```

Comment:
May be 3.5-dev only for critical fixes and dependency updates?

Reply:
I do agree, however I didn't think that was within scope for this change. I think that should be discussed and updated after the release of 3.5.8.